BggAlerter/Program.cs

```diff
@@ -17,47 +17,24 @@ static void Main(string[] args)
 
             RunApplication().Wait();
         }
-        public record Gamer(int Position, string Name, int Year, double Rating);
         private static async Task RunApplication()
         {
             // Register components
             var bggCaller = new BggCaller();
             var bggResponseParser = new BggResponseParser();
-
+            var filePath = @"C:\Users\afig3\Documents\test.csv";
+            //var bggGameReader = new BggFileReader();
+            //var oldGameList = bggGameReader.ReadBggGames(filePath);
+            //Console.WriteLine(oldGameList);
             // TODO: check database for existing entries, load to memory if any
-
-            using var streamReader = File.OpenText(@"C:\test.csv");
-            using var csvReader = new CsvReader(streamReader, CultureInfo.CurrentCulture);
-
-
-            var oldGL = new List<Game>;
-            var oldg = new Game();
-            var oldg = File.ReadLines(@"C:\test.csv")
-                           .Select(line => line.Split(','))
-                           .Select(tokens => new Game { Position = tokens[0], Name = tokens[1] })
-                           .ToList();
-
             var httpResponse = await bggCaller.GetBggResponse();
             var gameList = bggResponseParser.ParseBggResponse(httpResponse);
 
-            // TODO: save game list to the database
-
-            using var sw = new StreamWriter(@"C:\test.csv");
-            using var csvWriter = new CsvWriter(sw, CultureInfo.InvariantCulture);
-
-            foreach (var game in gameList) 
-            {
-                   {
-                        csvWriter.WriteField(game.Position);
-                        csvWriter.WriteField(game.Name);
-                        csvWriter.WriteField(game.Year);
-                        csvWriter.WriteField(game.Rating);
-                        csvWriter.NextRecord();
-                    }
-                    
-            }
-
+            var bggFileWriter = new BggFileWriter();
```

Comment:
move this to line24 - all class instance registrations is done there - to keep code style consistency

Reply:
still not done